ivy/functional/ivy/gradients.py

```diff
@@ -809,7 +809,125 @@ def lamb_update(
     Returns
     -------
     ret
-        The new function weights ws_new, following the LARS updates.
+        The new function weights ws_new, following the lamb updates.
+
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` inputs:
+
+    >>> w = ivy.array([1., 2, 3])
+    >>> dcdw = ivy.array([0.5,0.2,0.1])
+    >>> lr = ivy.array(0.1)
+    >>> vw_tm1 = ivy.zeros(1)
+    >>> mw_tm1 = ivy.zeros(3)
+    >>> step = ivy.array(1)
+    >>> new_weights = ivy.lamb_update(w,dcdw,lr,mw_tm1,vw_tm1,step)
+    >>> print(new_weights)
+    (ivy.array([0.784, 1.78 , 2.78 ]), ivy.array([0.05, 0.02, 0.01]), ivy.array([2.5e-04, 4.0e-05, 1.0e-05]))
+
+
+    >>> w = ivy.array([[1., 2, 3],[4, 6, 1],[1, 0, 7]])
+    >>> dcdw = ivy.array([[0.5, 0.2, 0.1],[0.3, 0.6, 0.4],[0.4, 0.7, 0.2]])
+    >>> lr = ivy.array(0.1)
+    >>> mw_tm1 = ivy.zeros((3,3))
+    >>> vw_tm1 = ivy.zeros(3)
+    >>> step = ivy.array(1)
+    >>> beta1=0.9,
+    >>> beta2=0.999,
+    >>> epsilon=1e-7,
+    >>> max_trust_ratio=10,
+    >>> decay_lambda=0,
+    >>> inplace=None,
```

Comment:
For the variables `beta1`,`beta2`,`epsilon`,`max_trust_ratio`,`decay_lambda`, and `inplace`, there exists an extra ',' that causes them to be initialized as tuples instead of floats and ints. Can you remove the extra ','? For example. `beta1=0.9` instead of `beta1=0.9,` Thanks!

Reply:
Please I am done with the changes requested.